source/NVDAObjects/behaviors.py

```diff
@@ -265,15 +265,40 @@ def event_textChange(self):
 		"""
 		self._event.set()
 
-	def _getTextLines(self):
-		"""Retrieve the text of this object in lines.
+	def _get_diffAlgo(self):
+		"""
+			This property controls which diff algorithm is used. The default
+			implementation returns either diffHandler.dmp or diffHandler.difflib
+			based on user preference. Subclasses can override this property to
+			choose a diffAlgo object (overriding user preference)
+			if one is incompatible with a particular application.
+			As of NVDA 2020.4, diffHandler.dmp is experimental. Therefore,
+			subclasses should either use the base implementation to check the
+			user config, or return diffHandler.difflib
+			to forcibly use Difflib.

```

Comment:
This coment would be better with a centralised check eg in `getDmpDiffAlgo`. Rather than saying "user preference" though it is clearer to say "if the user allows it"